edl_controller.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: training-job-controller
+  namespace: paddlecloud
```

Comment:
do we need to create the namespace before this yaml can claim?

Reply:
Indeed. We should update the documents later.